src/AzureDeprecation.Integrations.GitHub/Repositories/GitHubRepository.cs

```diff
@@ -35,7 +35,7 @@ public async Task PostCommentAsync(int issueNumber, string commentText)
 
         public async Task LockIssueAsync(int issueNumber)
         {
-            await _githubClient.Issue.Lock(_repoOwner, _repoName, issueNumber);
+            await _githubClient.Issue.LockUnlock.Lock(_repoOwner, _repoName, issueNumber);
```

Comment:
Can you elaborate on this change please? Or is it because of updated dependency?

Reply:
Yes, the method has moved in the Octokit API:
https://github.com/octokit/octokit.net/blob/063e85e4db5ad54ba4b121fb63ee86502247afee/Octokit/Clients/IIssuesClient.cs#L44
https://github.com/octokit/octokit.net/blob/main/Octokit/Clients/ILockUnlockClient.cs